<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home_DevWid</class>
 <widget class="QWidget" name="Home_DevWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>666</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_1">
     <property name="title">
      <string>常用操作</string>
     </property>
     <widget class="QPushButton" name="logBtn">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>60</y>
        <width>100</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>清除日志</string>
      </property>
     </widget>
     <widget class="QPushButton" name="eleClearBtn">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>60</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>电能清除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="factoryBtn">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>110</y>
        <width>101</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>恢复出厂</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cascadeBtn">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>110</y>
        <width>100</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>开启级联</string>
      </property>
     </widget>
     <widget class="QPushButton" name="closeAllBtn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>101</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>全 关</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enDelayBtn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>100</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>开关延时1S</string>
      </property>
     </widget>
     <widget class="QPushButton" name="disDelayBtn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>100</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>清除延时0S</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openAllBtn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>全 开</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ModbusRtu</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>110</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>传感器盒子</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="rtuEnBtn">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>60</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>开启</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rtuDisenBtn">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>60</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
     <widget class="QPushButton" name="boxDisenBtn">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>110</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bosEnBtn">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>110</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>开启</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>额定值配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QSpinBox" name="lineVolBox">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string>V</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>600</number>
        </property>
        <property name="value">
         <number>220</number>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QDoubleSpinBox" name="loopPowBox">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string>KW</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="value">
         <double>3.520000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="lineCurBox">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string>A</string>
        </property>
        <property name="value">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>回路电流额定值：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>相电压额定值：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QDoubleSpinBox" name="linePowBox">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="suffix">
         <string>KW</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="value">
         <double>7.040000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>回路功率额定值：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="loopVolBox">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string>V</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="maximum">
         <number>600</number>
        </property>
        <property name="value">
         <number>220</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>回路电压额定值：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>相电流额定值：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="loopCurBox">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string>A</string>
        </property>
        <property name="value">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>相功率额定值：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>设备配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>软件版本号</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fwEdit">
          <property name="maximumSize">
           <size>
            <width>138</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>设备类型</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="typeEdit">
          <property name="maximumSize">
           <size>
            <width>138</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>MPDU-Pro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>传感器盒子</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="sensorBox">
          <item>
           <property name="text">
            <string>没有</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>存在</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>设备系列：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="devSpecBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>3</number>
          </property>
          <item>
           <property name="text">
            <string>A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>C</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>D</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>相位数：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="lineNumBox">
          <item>
           <property name="text">
            <string>单相</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>三相</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>屏幕：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QComboBox" name="vhBox">
          <item>
           <property name="text">
            <string>垂直</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>水平</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>版本类型：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QComboBox" name="standardBox">
          <item>
           <property name="text">
            <string>标准</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>中性</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>定制</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>回路数量：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="loopNumBox">
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>断路器</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="breakerBox">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>没有</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>存在</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>语言选择：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="languageBox">
          <item>
           <property name="text">
            <string>中文</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>英文</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
